src/components/home/modals/CancelLease.vue

```diff
@@ -209,6 +206,45 @@ export default {
                 }
             }
             return addrInfo
+        },
+        showAmount() {
+            return browser.bigNumberFormatter(BigNumber(this.amount).dividedBy(1e8))
+        },
+        leaseId() {
+            return this.dataObject.stored_tx.txId
+        },
+        showAddress() {
+            let acc = new Account(NETWORK_BYTE)
+            let address = acc.convertPublicKeyToAddress(this.dataObject.stored_tx.senderPublicKey, NETWORK_BYTE)
```

Comment:
same related problems here. why here `this.dataObject.stored_tx.senderPublicKey` is always accessible? but in the previous case, we need extra judgment?

Reply:
In this case `this.dataObject` is initialized by props variable `coldPublicKey`. In previous one, it is  initialized by `coldAddressInfo` and it will return "" when there is no coldWallet.

Reply:
but coldPublicKey is also defaulted with ""? moreover, it seems that we will not judge the format of publicKey input by the user(i am not sure here). in those cases, this. coldPublicKey may have value(can be accessed) but incorrect?

Reply:
In new commit, cold wallet publicKey will be checked before going to confirm page so that we can avoid this problem and keep the data consistent throughout the process.